pom.xml

```diff
@@ -26,8 +26,8 @@
   </developers>
 
   <properties>
-    <maven-plugin-tools.version>3.6.4</maven-plugin-tools.version>
-    <maven.version>3.8.6</maven.version>
+    <maven-plugin-tools.version>3.9.0</maven-plugin-tools.version>
+    <maven.version>3.9.2</maven.version>
```

Comment:
Inconsistent with:

- https://github.com/jenkinsci/bridge-method-injector/pull/52
- https://github.com/jenkinsci/incrementals-tools/pull/47
- https://github.com/jenkinsci/lib-access-modifier/pull/144
- https://github.com/jenkinsci/maven-hpi-plugin/pull/440
- https://github.com/jenkinsci/maven-jellydoc-plugin/pull/34
- https://github.com/jenkinsci/maven-stapler-plugin/pull/40

Please fix this by moving to a Maven baseline of 3.8.1 as in these other PRs.